cranelift/filetests/filetests/isa/s390x/bitops.clif

```diff
@@ -36,34 +36,36 @@ block0(v0: i64):
 }
 
 ; block0:
-;   llihf %r5, 2863311530
-;   iilf %r5, 2863311530
-;   sllg %r4, %r2, 1
-;   srlg %r2, %r2, 1
-;   ngr %r4, %r5
-;   xilf %r5, 4294967295
-;   xihf %r5, 4294967295
-;   ngrk %r5, %r2, %r5
-;   ogrk %r2, %r4, %r5
-;   llihf %r4, 3435973836
-;   iilf %r4, 3435973836
-;   sllg %r3, %r2, 2
-;   srlg %r5, %r2, 2
-;   ngr %r3, %r4
-;   xilf %r4, 4294967295
-;   xihf %r4, 4294967295
-;   ngrk %r4, %r5, %r4
-;   ogrk %r5, %r3, %r4
-;   llihf %r3, 4042322160
-;   iilf %r3, 4042322160
-;   sllg %r2, %r5, 4
-;   srlg %r4, %r5, 4
-;   ngr %r2, %r3
+;   lgr %r3, %r2
+;   llihf %r2, 2863311530
+;   iilf %r2, 2863311530
+;   lgr %r5, %r3
```

Comment:
These two `lgr` look new - this is why the new code is two instructions longer than the old code.  Not sure if this is something that could still be improved in regalloc, or if this is just one of those random changes ...   In any case, not a big deal, I just wanted to point it out.

Reply:
I briefly looked but it was nothing really obvious. It's possible that my changes in bytecodealliance/regalloc2#74 might help a bit, but I'm not sure; let's see if it reverts back once I update tests there with this merged :-)